frontend/src/i18n/locales/de/notifications.json

```diff
@@ -0,0 +1,14 @@
+{
+  "You don't have any notifications right now": "Sie haben im Moment keine Benachrichtigungen",
+  "No message": "Keine Nachricht",
+  "Mark as read": "Als gelesen markieren",
+  "Show notifications": "Benachrichtigungen anzeigen",
+  "You have unread notifications": "Sie haben ungelesene Benachrichtigungen",
+  "Notifications": "Benachrichtigungen",
+  "Mark all as read": "Alle als gelesen markieren",
+  "Clear": "Klar",
+  "View all notifications": "Alle Benachrichtigungen anzeigen",
```

Comment:
Alle Meldungen anzeigen

Reply:
Nachrichten (we use that term for e-mail too).